library/cwm/src/modules/CWM.rb

```diff
@@ -917,8 +919,11 @@ def SetValidationFailedHandler(handler)
     # @param [String] back_button label for dialog back button
     # @param [String] next_button label for dialog next button
     # @param [String] abort_button label for dialog abort button
+    # @param [Array] skip_store_for list of events that skip storring of widgets.
+    #   Useful mainly when some widget return event that skip storring like
+    #   "reset" button or redrawing without storing
```

Comment:
list of events for which the value of the widget will not be stored.
Useful mainly when some widget returns an event that should not trigger the storing, like a reset button or a redrawing